src/st/no3/3C4EC.c

```diff
@@ -20,7 +21,8 @@ void EntityPushAlucard(Entity* entity) {
     case 1:
         player->posX.val += 0x60000;
         if (roomLayout->unkA > 0x800) {
-            g_Entities[UNK_ENTITY_1].ext.generic.unk7C.S8.unk0 = 0;
+            g_Entities[alucardController].ext.alucardController.unk7C =
```

Comment:
Why do you use `g_Entities[UNK_ENTITY_1]` and `g_Entities[alucardController]`?

Reply:
Sorry, that was a mistake, i'll take care of it.

Reply:
Why is using a local variable better than a define? I see there's many usages of `g_Entities[1] across the codebase.

Reply:
You have a point there, corrected.